src/main/webapp/place-suggestions.html

```diff
@@ -0,0 +1,131 @@
+    <title>Overnightly | Plan For Me</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/constant-style.css" />
+    <link rel="stylesheet" href="styles/place-suggestions.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/favicon/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/favicon/favicon-16x16.png"
+    />
+    <link
+      rel="mask-icon"
+      href="/favicon/safari-pinned-tab.svg"
+      color="#5bbad5"
+    />
+    <link rel="shortcut icon" href="/favicon/favicon.ico" />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="msapplication-config" content="/favicon/browserconfig.xml" />
+    <meta name="theme-color" content="#ffffff" />
+  </head>
+  <body onload="init();">
+    <nav class="blue darken-2" role="navigation">
+      <div class="nav-wrapper container">
+        <a href="index.html" class="brand-logo">Overnightly</a>
+        <a href="#" data-target="mobile-demo" class="sidenav-trigger">
+          <i class="material-icons">menu</i>
+        </a>
+        <ul class="right hide-on-med-and-down" id="menu-links">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="my-trips.html">My Trips</a></li>
+          <li class="active"><a href="#">Plan For Me</a></li>
+          <li><a href="trips-network.html">Trips Network</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <ul class="sidenav" id="mobile-demo">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="my-trips.html">My Trips</a></li>
+      <li class="active"><a href="#">Plan For Me</a></li>
+      <li><a href="trips-network.html">Trips Network</a></li>
+    </ul>
+
+    <div id="map"></div>
+
+    <div class="page-container container">
+      <div id="input-container">
+        <div id="city-input-area" class="row">
+          <h5 id="city-input-title" class="col s12 m7">
+            What city do you want to visit?
+          </h5>
+          <div id="city-input-field" class="input-field col s12 m5">
+            <input placeholder="Enter a city" id="city-input" type="text">
+          </div>
+        </div>
+
+        <div id="filter-selection-area" class="row">
+          <h5 id="filter-selection-title" class="col s12 m7">
+            What type of place do you want to visit?
+          </h5>
+          <div id="filter-selection" class="input-field col s12 m5">
+            <select id="filters">
+              <option value="" disabled selected>Choose place type</option>
+              <option value="amusement_park">Amusement Park</option>
```

Comment:
Are we hardcoding this, or will we be able to pull what kind of places we can visit based on which city the user wants to go?

Reply:
These are based on the [available place types in the Places API for the nearby search feature](https://developers.google.com/places/supported_types), from which I selected what I thought were most relevant for most trip planning. I figured this should cover all the bases for any city they want to visit, though I'm not quite satisfied with the lack of more specific categories like "italian restaurant" instead of just "restaurant". @chaowangzj mentioned using the gcid string which has more specific categories, but @xiangchengz thought this might be an internal feature. My only solution right now is building a simple web crawler that categorizes locations for us, though I still have to look into how to implement that.